dspace-server-webapp/src/test/java/org/dspace/app/rest/StatisticsRestRepositoryIT.java

```diff
@@ -534,14 +658,43 @@ public void TotalDownloadsReport_Bitstream() throws Exception {
         expectedPoint.setId(bitstreamVisited.getID().toString());
 
         // And request that bitstreams's TotalDownloads stat report
-        getClient().perform(
+        getClient(adminToken).perform(
             get("/api/statistics/usagereports/" + bitstreamVisited.getID() + "_" + TOTAL_DOWNLOADS_REPORT_ID))
                    // ** THEN **
                    .andExpect(status().isOk())
                    .andExpect(jsonPath("$", Matchers.is(
                        UsageReportMatcher
                            .matchUsageReport(bitstreamVisited.getID() + "_" + TOTAL_DOWNLOADS_REPORT_ID,
                                TOTAL_DOWNLOADS_REPORT_ID, Arrays.asList(expectedPoint)))));
+
+        // only admin has access to downloads report
+        getClient(loggedInToken).perform(
+                  get("/api/statistics/usagereports/" + bitstreamVisited.getID() + "_" + TOTAL_DOWNLOADS_REPORT_ID))
+                 .andExpect(status().isForbidden());
+
+        getClient().perform(
+                  get("/api/statistics/usagereports/" + bitstreamVisited.getID() + "_" + TOTAL_DOWNLOADS_REPORT_ID))
+                 .andExpect(status().isUnauthorized());
+
+        // make statistics visible to all
+        configurationService.setProperty("usage-statistics.authorization.admin.usage", false);
+
+        // only admin has access to downloads report
```

Comment:
This comment is incorrect & can be removed. You are testing everyone can now access the report